components/dashboard/src/data/organizations/create-org-mutation.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { Organization } from "@gitpod/gitpod-protocol";
+import { useMutation } from "@tanstack/react-query";
+import { useOrganizationsInvalidator } from "./orgs-query";
+import { publicApiTeamToProtocol, teamsService } from "../../service/public-api";
+
+type UpdateOrgArgs = Pick<Organization, "name" | "slug">;
```

Comment:
The name should be CreateOrgArgs, but we only need a name, so I think we can just use `string`